lib/ansible/modules/cloud/vmware/vmware_guest_powerstate.py

```diff
@@ -59,6 +59,13 @@
     - '   folder: vm/folder2'
     - '   folder: folder2'
     default: /vm
+  wait_for_state_change:
+    description:
+    - If the C(state) is set to C(shutdownguest), by default the module will return immediately after sending the shutdown signal.
```

Comment:
perhaps this should be "C(shutdown-guest)" to match the "choices" on line 31 above?